openslides/users/static/js/users/users.js

```diff
@@ -18,7 +18,48 @@ angular.module('OpenSlidesApp.users', [])
                 }
             }
         })
-        .state('users.user.create', {})
+        .state('users.user.create', {
+            resolve: {
+                groups: function(Group) {
+                    return Group.findAll();
+                }
+            }
+        })
+        .state('users.user.csv-import', {
```

Comment:
Please move this state and the group state below the users stats after the rebase.